### PR TITLE
Rebrand adapting imports

### PR DESCRIPTION
**Pull request by AdoAdoAdo**
## Reasoning behind the pull request
- adapt imports to new org
- module name change to follow new org
  
## Proposed changes
- use latest tags from all dependencies to new org
- change name to reflect new organization

## Testing procedure
- run all system tests

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: testscommon/state/accountWrapperMock.proto [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+package proto;
+
+option go_package = "state";
+option (gogoproto.stable_marshaler_all) = true;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+message AccountWrapMockData {
+  int64 MockValue = 1 [(gogoproto.jsontag) = "MockValue"];
+}
```
Comment (iulianpascalau) [inline testscommon/state/accountWrapperMock.proto:12]: Empty line?

File: testscommon/state/accountWrapperMock.pb.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ var _ = math.Inf
 const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
 type AccountWrapMockData struct {
-	MockValue int64 `protobuf:"varint,1,opt,name=MockValue,proto3" json:"MockValue,omitempty"`
+	MockValue int64 `protobuf:"varint,1,opt,name=MockValue,proto3" json:"MockValue"`
```
Comment (iulianpascalau) [inline testscommon/state/accountWrapperMock.pb.go:29]: This previously had an `omitempty` json tag. However I don't feel this is important. Can remain like this.

**Review by iulianpascalau — COMMENTED** [review]
GJ :+1: 
just some minor stuff

File: go.mod [hunk 3]
```diff
@@ -24,6 +14,16 @@ require (
...
 	github.com/gorilla/websocket v1.5.0
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/multiversx/mx-chain-core-go v1.1.30
+	github.com/multiversx/mx-chain-crypto-go v1.2.5
+	github.com/multiversx/mx-chain-es-indexer-go v1.3.7
+	github.com/multiversx/mx-chain-logger-go v1.0.11
+	github.com/multiversx/mx-chain-p2p-go v1.0.10
+	github.com/multiversx/mx-chain-storage-go v1.0.7
+	github.com/multiversx/mx-chain-vm-common-go v1.3.34
+	github.com/multiversx/mx-chain-vm-v1_2-go v1.2.49-0.20230110142521-9b4e38803f3b
```
[the remark this answers is not in the document]
Comment (AdoAdoAdo) [reply go.mod:24]: done

File: testscommon/state/accountWrapperMock.proto (hunk 1 above)
Comment (AdoAdoAdo) [reply testscommon/state/accountWrapperMock.proto:12]: done

File: testscommon/state/accountWrapperMock.pb.go (hunk 2 above)
Comment (AdoAdoAdo) [reply testscommon/state/accountWrapperMock.pb.go:29]: left it as is, there was no test scenario for omit empty.

File: examples/messageSign_test.go [hunk 4]
```diff
@@ -30,7 +30,7 @@ const elrondSignedMessagePrefix = "\x17Elrond Signed Message:\n"
 var messageSigningHasher = keccak.NewKeccak()
 
 func TestVerifyMessageSignatureFromLedger(t *testing.T) {
-	// these field values were obtained by using Elrond App for Ledger
+	// these field values were obtained by using MultiversX App for Ledger
```
[the remark this answers is not in the document]
Comment (AdoAdoAdo) [reply examples/messageSign_test.go:33]: renamed the variable to signedMessagePrefix

**Review by SebastianMarian — COMMENTED** [review]
Also file testscommon/state/proto/accountWrapperMock.proto  is empty. Do we still need it?

File: epochStart/bootstrap/startInEpochScheduled_test.go [hunk 5]
```diff
@@ -6,19 +6,19 @@ import (
 	"math/big"
 	"testing"
 
-	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/multiversx/mx-chain-go/process"
```
[the remark this answers is not in the document]
Comment (AdoAdoAdo) [reply epochStart/bootstrap/startInEpochScheduled_test.go:9]: done

File: scripts/testnet/variables.sh [hunk 6]
```diff
@@ -3,33 +3,33 @@
...
-# repository (elrond-txgen-go).
+# Enable the MultiversX Transaction Generator. Note that this is a private
+# repository (mx-chain-txgen-go).
 export USE_TXGEN=0
 
 # Path where the testnet will be instantiated. This folder is assumed to not
 # exist, but it doesn't matter if it already does. It will be created if not,
 # anyway.
-export TESTNETDIR="$HOME/Elrond/testnet"
+export TESTNETDIR="$HOME/Multiversx/testnet"
```
[the remark this answers is not in the document]
Comment (AdoAdoAdo) [reply scripts/testnet/variables.sh:20]: done

File: examples/messageSign_test.go [hunk 7]
```diff
@@ -25,7 +25,7 @@ import (
 */
 
 // This prefix should be added when computing the hash to be signed
-const elrondSignedMessagePrefix = "\x17Elrond Signed Message:\n"
+const signedMessagePrefix = "\x17Elrond Signed Message:\n"
```
Comment (SebastianMarian) [inline examples/messageSign_test.go:28]: We keep Elrond in string?

File: README.md [hunk 8]
```diff
@@ -30,26 +30,26 @@ The main branch that will be used is the master branch. Alternatively, an older
...
 
 ### Step 3: creating the node’s identity:
-In order to be registered in the Elrond Network, a node must possess 2 types of (secret key, public key) pairs. One is used to identify the node’s credential used to generate transactions (having the sender field its account address) and the other is used in the process of the block signing. Please note that this is a preliminary mechanism, in the next releases the first (private, public key) pair will be dropped when the staking mechanism will be fully implemented. To build and run the keygenerator, the following commands will need to be run:
+In order to be registered in the MultiversX network, a node must possess 2 types of (secret key, public key) pairs. One is used to identify the node’s credential used to generate transactions (having the sender field its account address) and the other is used in the process of the block signing. Please note that this is a preliminary mechanism, in the next releases the first (private, public key) pair will be dropped when the staking mechanism will be fully implemented. To build and run the keygenerator, the following commands will need to be run:
```
[the remark this answers is not in the document]
Comment (SebastianMarian) [reply README.md:42]: I just have seen that in L4 the change was from Elrond Network to MultiversX that I have asked it?

File: examples/messageSign_test.go (hunk 7 above)
Comment (iulianpascalau) [reply examples/messageSign_test.go:28]: for backwards compatibility, yes

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**